Connection/Handler.php

```diff
@@ -153,27 +153,27 @@ public function run()
         while (true) {
             foreach ($connection->select() as $node) {
 
-            // Connection has failed to detect the node, maybe it is a resource
-            // from a merged client in a server.
-            if (false === is_object($node)) {
-                $socket = $node;
+                // Connection has failed to detect the node, maybe it is a resource
```

Comment:
It's not useless. It was improperly indented (if at the same level as the foreach).